File: Codewars/Kata/8 Kyu/Sum without Highest and Lowest Number/sumArray.php
function sumArray($array) {
    if ($array == null || count($array) == 0 || count($array) == 1)
       { return 0;}
    $n = $m = $result = $array[0];
    for ($i = 1; $i < count($array); $i++)
    {
        $result = $result + $array[$i];
        if ($array[$i] < $m)
            $m = $array[$i];
        if ($array[$i] > $n)
            $n = $array[$i];
    }
    return $result - $m - $n;
}

File: Codewars/Kata/8 Kyu/Sum without Highest and Lowest Number/sum_array.py
def sum_array(arr):
    if (arr == None or len(arr) == 0 or len(arr) == 1) :
           return 0;
    n, m, result = arr[0],arr[0],0;
    for i in arr:
        if i < m:
            m = i;
        elif i > n:
            n = i;
        result = result + i;
    return result - m - n;
